src/services/export.service.ts

```diff
@@ -141,8 +142,12 @@ export class ExportService implements ExportServiceAbstraction {
 
         await Promise.all(promises);
 
+        const testValue = await this.cryptoService.randomNumber(2048, 65536);
```

Comment:
just curious, why a number? I was expecting Utils.NewGuid() or something

Reply:
Because that's a better idea and I didn't think of that ;)